package.json

```diff
@@ -173,9 +173,10 @@
     "mongodb": "^6.1.0",
     "mongodb-v4": "npm:mongodb@^4.16.0",
     "mongodb-v5": "npm:mongodb@^5.4.0",
-    "mongoose": "^7.1.0",
-    "mongoose-v5": "npm:mongoose@^5.13.17",
-    "mongoose-v6": "npm:mongoose@^6.11.1",
```

Comment:
Mongoose is still publishing releases for the old major versions e.g. see https://github.com/Automattic/mongoose/tree/5.13.22